.github/workflows/spider-pr.yaml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test-spider:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     env:
       AWS_KEY: QUtJQTRLMllUQzIzRzVZRkxENkg=
       AWS_SEC: b1RmNXNxc0VSNlNKdFNhbUdlcjJwZHR6YXQwNXpRRDRPZUtoVkpPSA==
```

Comment:
Is this an actual secret?

Reply:
Yes, it's an encoded secret. I do this so that Github Actions triggered by users submitting pull requests from their fork will still upload to S3. Github doesn't export secrets in these situations, and that's why the commenting fails.